config/fcos/v1_6_exp/validate.go

```diff
@@ -98,3 +98,20 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func (m Module) Validate(c path.ContextPath) (r report.Report) {
+	if m.Name == "" && m.Content == "" {
+		r.AddOnError(c.Append("name"), common.ErrSelinuxContentNotSpecified)
+		r.AddOnError(c.Append("content"), common.ErrSelinuxContentNotSpecified)
+	} else {
```

Comment:
I don't think we need this.

Reply:
I removed both this implementation and test.

Reply:
Thanks for your review :)